lib/i18n/default.js

```diff
@@ -114,15 +114,19 @@ function isGoodWord(word) {
     // filter out words that cannot be in the dataset,
     // because they would be either tokenized/preprocessed out or
     // they are unlikely to be used with voice
-    return true;
+    return ! /[\n\r\t"]/.test(word);
```

Comment:
Spaces (including \n, \r and the like) and quotes cannot be part of a word, so this test is redundant.

Reply:
reverted.